Makefile

```diff
@@ -56,6 +56,9 @@ vet:
 check_license:
 	./scripts/check_license.sh
 
+check_shell:
+	find . -type f -name *.sh -exec shellcheck {} +
```

Comment:
this is a good idea, but you'll need to have the `*.sh` surrounded by double-quotes. i.e. `"*.sh"`

Reply:
Will change.